openedx/features/enterprise_support/serializers.py

```diff
@@ -0,0 +1,44 @@
+"""
+Defines serializers for enterprise_support.
+"""
+
+
+from rest_framework import serializers
+
+try:
+    from enterprise.api.v1.serializers import \
+        EnterpriseCourseEnrollmentReadOnlySerializer as BaseEnterpriseCourseEnrollmentSerializer
+    from enterprise.models import EnterpriseCourseEnrollment
+except ImportError:  # pragma: no cover
+    pass
+
```

Comment:
also, shouldn't we log anything in except block?

Reply:
Imports can fail if enterprise isn't installed. We have not been logging these import errors since they wouldn't be notable in the instances where it isn't. More info on the intention here in this pr https://github.com/openedx/edx-platform/pull/14053/files.